frontends/precinct-scanner/src/screens/scan_warning_screen.tsx

```diff
@@ -118,26 +125,27 @@ function UndervoteWarningScreen({
         <h1>Review Your Ballot</h1>
         {blankContestNames.length > 0 && (
           <Text small>
-            You did not vote for {toSentence(blankContestNames, ', ', ' or ')}.
+            No votes detected for: {toSentence(blankContestNames, ', ', ' or ')}
+            .
           </Text>
         )}
         {partiallyVotedContestNames.length > 0 && (
           <Text small>
-            You can vote for more people for{' '}
+            You may vote for more candidates for:{' '}
             {toSentence(partiallyVotedContestNames, ', ', ' and ')}.
           </Text>
         )}
         <p>
-          <Button primary large onPress={() => setConfirmTabulate(true)}>
-            Count my ballot
+          <Button onPress={scanner.returnBallot}>Return Ballot</Button> or{' '}
+          <Button primary onPress={() => setConfirmTabulate(true)}>
+            Count Ballot As Is
           </Button>
         </p>
-        <p>
-          <Button small onPress={scanner.returnBallot}>
-            Return my ballot
-          </Button>
-        </p>
-        <Text italic>Ask a poll worker if you need help.</Text>
+        <Text italic>
+          Marking votes in every contest is not required.
```

Comment:
Maybe to make this more "plain," something like:

> You don't have to vote for every contest.

or 

> Your votes will be counted even if you leave some contests blank.

Reply:
Thanks. I'm leaning towards using two different phrases to say the same thing in two separate ways with the goal of ensuring that the message is communicated.

More active voice on the page:
> Your votes will count, even if you leave some contests blank.

Confirmation modal:
> You are not required to vote in every contest.

Reply:
Upon further editing, I'm leaning towards a slightly more generic phrase in both locations:

> Your votes will count, even if you leave some blank.